java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -498,19 +498,27 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
     }
 
     private static Object[] bitmapCommands(BaseTransaction<?> transaction) {
-        String key = "{key-" + UUID.randomUUID();
+        String key1 = "{bitmapKey}-1" + UUID.randomUUID();
+        String key2 = "{bitmapKey}-2" + UUID.randomUUID();
 
-        transaction.set(key, "foobar").bitcount(key).bitcount(key, 1, 1);
+        transaction
+                .set(key1, "foobar")
+                .bitcount(key1)
+                .bitcount(key1, 1, 1)
+                .setbit(key2, 1, 1)
```

Comment:
nit: why are you using a new key here?

Reply:
Ok - you know the first string is "foobar".  
But its your tests... I don't really care too much